samples/VK.gs

```diff
@@ -13,9 +13,9 @@ function run() {
   var service = getService();
   if (service.hasAccess()) {
     // GET requests require access_token parameter
-    var url = 'https://api.vk.com/method/groups.get?access_token=' +
-        service.getAccessToken();
+    var url = 'https://api.vk.com/method/users.get?version=5.89&fields=first_name,last_name,email&access_token=' + service.getAccessToken()
```

Comment:
If it isn't required, please don't hard-code the version. Having the version in the URL makes this sample fall out of date faster.

Reply:
ok!👍